pyglossary/entry_filters.py

```diff
@@ -258,7 +258,9 @@ class SkipDataEntry(EntryFilter):
 	desc = "Skip resources / data files"
 
 	def run(self, entry: "EntryType") -> "EntryType | None":
-		if entry.isData():
+		if entry.isData() and entry.b_word.endswith(
+				(b'.ogg', b'.mp3', b'.mp4', b'.wav', b'.spx', b'.mov', b'.pdf', b'.jpg', b'.jpeg' ),
```

Comment:
Looks like these files are not recognized as Data Entry by the Reader class. This change should not be needed.

Reply:
reverted to `master`; in fact I was not 100% sure it needed to be included; thanks!

Reply:
also reverted the dsl/lex.py